test/__fixtures/node-proxy/proxy.js

```diff
@@ -0,0 +1,29 @@
+const HttpProxy = require("http-proxy");
+const program = require("commander");
+const fs = require("fs");
+
+program
+  .option("-h, --host [host]", "Host")
+  .option("-p, --port [port]", "Port")
+  .option("-l, --listen [listen]", "Listen port")
+  .option("-s, --secure [secure]", "Secure")
+  .parse(process.argv);
+
+let host = program.host || 'localhost';
+let port = program.port || 55000;
+let listen = program.listen || 11235;
+let secure = program.secure || false;
+
+const proxyConfig = {target:`ws://${host}:${port}`, ws: true, xfwd: true};
```

Comment:
Note the xfwd:true - that is what you need to implement on the secure gateway

Reply:
Noted thanks!